apps/src/netsim/NetSimTable.js

```diff
@@ -523,7 +523,7 @@ NetSimTable.prototype.updateCacheRow_ = function(id, row) {
 
   if (!_.isEqual(oldRow, newRow)) {
     this.cache_[id] = newRow;
-    this.tableChange.notifyObservers();
+    this.tableChange.notifyObservers('updateCacheRow');
```

Comment:
I'm not _thrilled_ about passing the change type along in this event - it feels like we're giving observers the opportunity/reason to care about the internal implementation of the the simulation table.  Maybe we can achieve the optimization you're looking for (not listening for message deletes) another way?